src/main/java/com/snack/news/domain/tag/Tag.java

```diff
@@ -1,10 +1,7 @@
 package com.snack.news.domain.tag;
 
 
-import lombok.Builder;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.ToString;
+import lombok.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'lombok'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)